portal-kernel/src/com/liferay/portal/kernel/workflow/BaseWorkflowHandler.java

```diff
@@ -155,12 +156,17 @@ public String getURLEditWorkflowTask(
 		throws PortalException {
 
 		try {
-			PortletURL portletURL = PortalUtil.getControlPanelPortletURL(
+			PortletURL portletURL = PortletURLFactoryUtil.create(
 				serviceContext.getRequest(), PortletKeys.MY_WORKFLOW_TASK,
 				PortletRequest.RENDER_PHASE);
 
+			if (portletURL == null) {
```

Comment:
Hey @robertoDiaz,

Is it possible that portletURL is null here?

Thanks!

Reply:
Hey @inacionery 

You are right , is not possible, an instance is always been returned.

I'll update the PR